components/suggestion/command_provider/app_command_parser.ts

```diff
@@ -205,19 +713,19 @@ export class AppCommandParser {
             const res = await this.store.dispatch(doAppCall(payload)) as {data?: AppCallResponse; error?: Error};
             if (res.error) {
                 this.displayError(res.error);
-                return null;
+                return undefined;
             }
             callResponse = res.data;
         } catch (e) {
             this.displayError(e);
-            return null;
+            return undefined;
         }
 
-        if (callResponse && callResponse.form) {
+        if (callResponse?.form) {
```

Comment:
We can return directly here `callResponse?.form`, since in case it is not defined, it will return undefined.